packages/entando-apimanager/modules/api/APIProvider.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { config } from './apiManager';
+import { setApi } from '../state/api/actions';
+
+class APIProvider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.initApiManager();
+  }
+
+  initApiManager() {
+    const {
+      store,
+      plugins,
+      domain,
+      mockMode,
+      onLogout,
+      onLogin,
+    } = this.props;
+
+    config(store, onLogout, onLogin);
+
+    store.dispatch(setApi({
+      domain,
+      useMocks: mockMode,
+    }));
+
+    if (plugins.length) {
+      plugins.forEach((plugin) => {
+        if (plugin.apiManagerConfig) {
+          plugin.apiManagerConfig(store, onLogout, onLogin);
+        }
+      });
+    }
+  }
+
+  render() {
+    return (
+      <React.Fragment>{this.props.children}</React.Fragment>
+    );
+  }
+}
+
+APIProvider.propTypes = {
+  store: PropTypes.shape({}).isRequired,
+  onLogin: PropTypes.func.isRequired,
+  onLogout: PropTypes.func.isRequired,
+  domain: PropTypes.string.isRequired,
+  plugins: PropTypes.arrayOf(PropTypes.shape({})),
+  mockMode: PropTypes.bool,
```

Comment:
as a rule booleans should always be named as questions, to make it easier to understand that it just accept `true` or `false` on top of that we should always avoid renaming props if not necessary, it just generates confusion and it is not crystal clear what the var is used for.

therefore, just keep the name `useMocks`

Reply:
done